system-tests/tests/smoke/cre/por_test.go

```diff
@@ -485,6 +571,32 @@ func setupPoRTestEnvironment(
 	require.NoError(t, err, "failed to convert chain ID to int")
 	chainIDUint64 := libc.MustSafeUint64(int64(chainIDInt))
 
+	jobSpecFactoryFuncs := []keystonetypes.JobSpecFactoryFn{
+		creconsensus.ConsensusJobSpecFactoryFn(chainIDUint64),
+		crecron.CronJobSpecFactoryFn(cronBinaryPathInTheContainer),
+		cregateway.GatewayJobSpecFactoryFn(extraAllowedPorts, []string{}, []string{"0.0.0.0/0"}),
+		crecompute.ComputeJobSpecFactoryFn,
+	}
+
+	if rcti != nil {
+		var readContractBinaryPathInTheContainer string
+		if in.DependenciesConfig.CronCapabilityBinaryPath != "" {
```

Comment:
If "cron" binary path is not empty, we are replacing it with read capability; if it's empty, we are using "cron"? What's the connection between them? Maybe we should have a separate piece of code selecting read capability from a container or a local path to copy?

Reply:
I think it's just a typo, should just be:
```
if in.DependenciesConfig.ReadContractCapabilityBinaryPath != "" {
```